### PR TITLE
docs: update mnist CNN

### PR DESCRIPTION
**Pull request by jfrery**

### CODE REVIEW

File: docs/advanced_examples/ConvolutionalNeuralNetwork.ipynb [hunk 1]
```diff
@@ -488,146 +412,47 @@
...
+   "execution_count": 8,
    "id": "6e8b6471",
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Keygen time: 137.80s\n"
+      "Keygen time: 3.98s\n"
```
Comment (RomanBredehoft) [inline docs/advanced_examples/ConvolutionalNeuralNetwork.ipynb:448]: 😍

File: docs/advanced_examples/ConvolutionalNeuralNetwork.ipynb [hunk 2]
```diff
@@ -649,14 +474,14 @@
...
-      "100%|██████████| 1/1 [00:32<00:00, 32.33s/it]"
+      "100%|██████████| 100/100 [04:19<00:00,  2.59s/it]"
      ]
     },
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Time per inference in FHE: 32.34\n"
+      "Time per inference in FHE: 2.59 with 99.00% accuracy\n"
```
Comment (RomanBredehoft) [inline docs/advanced_examples/ConvolutionalNeuralNetwork.ipynb:484]: 😍

File: docs/advanced_examples/ConvolutionalNeuralNetwork.ipynb [hunk 3]
```diff
@@ -688,13 +515,9 @@
    "source": [
     "### Conclusion\n",
     "\n",
-    "We see that quantization with **3** bit weight and activations is the best viable FHE configuration,\n",
-    "as the accumulator bit width for this configuration is between **7 and 8** bits (can vary due to the final \n",
-    "distribution of the weights). The accuracy in this setting, 92% is a few percentage points \n",
-    "under the maximum accuracy achievable with larger accumulator bit widths (97-98%). \n",
+    "In this example, a simple CNN model is trained with torch and reach 98% accuracy in clear. The model is then converted to FHE and evaluated over 100 samples in FHE.\n",
```
Comment (RomanBredehoft) [inline docs/advanced_examples/ConvolutionalNeuralNetwork.ipynb:518]: 99 right ?

Comment (jfrery) [reply docs/advanced_examples/ConvolutionalNeuralNetwork.ipynb:518]: The torch model over the entire test set is 98... and the model in fhe over a subset from the test of 100 samples is 99%. I did not mention the accuracy but just say it's similar. Not sure what's best.

**Review by fd0r — APPROVED** [review]

File: docs/advanced_examples/ConvolutionalNeuralNetwork.ipynb (hunk 3 above)
Comment (RomanBredehoft) [reply docs/advanced_examples/ConvolutionalNeuralNetwork.ipynb:518]: ok fine I guess

**Review by andrei-stoian-zama — APPROVED** [review]

**Review by fd0r — APPROVED** [review]

**Review by RomanBredehoft — APPROVED** [review]

**Pull request merged**